helm/ngrok-operator/templates/controller-deployment.yaml

```diff
@@ -62,6 +62,7 @@ spec:
         - /api-manager
         args:
         - --release-name={{ .Release.Name }}
+        - --cluster-id={{ .Values.clusterId | required "Missing required .Values.clusterId!"}}
```

Comment:
I think we need a way to have this not be supplied for an initial install. The requirements we talked about in slack are detained a bit better by this old issue where we resolved the issue that we required an authtoken and API key on start https://github.com/ngrok/ngrok-operator/issues/317

Reply:
I'm reading from that issue that the desire is the ngrok-operator can be installed into a non-functional/non-reconciling state and we give an error event on the `kind: KubernetesOperator` itself.

Is that what you mean?

Reply:
yep, i think something like that would fufill the requirements. Similarly, if you one click install, the operator is just going to log and event that it needs the credentias I believe. The idea is a helm instal without custom options should "just work" to at least helm install, but it may not fully function.